fundingmanager.go

```diff
@@ -3144,6 +3275,17 @@ func (f *fundingManager) handleErrorMsg(fmsg *fundingErrorMsg) {
 	)
 	fndgLog.Errorf(fundingErr.Error())
 
+	// If this was a PSBT funding flow, the remote likely timed out because
+	// we waited too long. Return a nice error message to the user in that
+	// case so the user knows what's the problem.
+	if resCtx.reservation.IsPsbt() {
```

Comment:
You're right, the original error shouldn't be suppressed. I just wrapped the error with the "remote canceled, possibly timed out" message instead of replacing it.